README.md

```diff
@@ -2,24 +2,19 @@
 
 Python script to parse the output of "virsh dumpxml" command captured with sosreport
```

Comment:
Can someone remove those double quotes and user backtick instead?

Python script to parse the output of `virsh dumpxml` command captured with sosreport